Find a functionally equivalent code:com.android.internal.policy.impl.PhoneWindowManager.canHideNavigationBar:COMMENT
Method Modifier: private     internal    
Comment:/**
 * @return whether the navigation bar can be hidden, e.g. the device has a
 * navigation bar and touch exploration is not enabled
 */

@@ -1,3 +1,3 @@
 {
-    return mHasNavigationBar && !mTouchExplorationEnabled;
+    return mHasNavigationBar && !mAccessibilityManager.isTouchExplorationEnabled();
 }

